src/components/accordion/Accordions.tsx

```diff
@@ -0,0 +1,98 @@
+import { FC, ReactNode } from 'react'
+import { useTranslation } from 'react-i18next'
+
+import Typography from '@mui/material/Typography'
+import Box from '@mui/material/Box'
+import Accordion, { AccordionProps } from '@mui/material/Accordion'
+import AccordionSummary from '@mui/material/AccordionSummary'
+import AccordionDetails from '@mui/material/AccordionDetails'
+
+import { styles } from '~/components/accordion/Accordion.styles'
+
+import { AccordionItem } from '~/types'
+import { Variant } from '@mui/material/styles/createTypography'
+
+interface Sx {
+  [key: string]: any
```

Comment:
if we know keys which can be in this sx we should provide them instead of key:string

Reply:
don't you really think that it is unnecessary to type styles?

Reply:
no, it will be helpful for those who will use your component, they will know which keys in the object they can provide

Reply:
it would be helpful if we have required styles for Accordions but there are no required fields besides noShowMoreIcon and withShowMoreIcon, nested fields with inside both of them are just by your choice and we can't know if in the future those who will be using this component will need to pass all styles that are in accordion

Reply:
so noShowMoreIcon and withShowMoreIcon should be described in types  if they are required for now

Reply:
typed